dev/com.ibm.ws.concurrent.mp.1.0/src/com/ibm/ws/concurrent/mp/service/ContextServiceImpl.java

```diff
@@ -41,17 +41,8 @@
 import com.ibm.websphere.ras.TraceComponent;
 import com.ibm.websphere.ras.annotation.Trivial;
 import com.ibm.ws.concurrent.ContextualAction;
-import com.ibm.ws.concurrent.mp.ContextualBiConsumer;
-import com.ibm.ws.concurrent.mp.ContextualBiFunction;
-import com.ibm.ws.concurrent.mp.ContextualCallable;
-import com.ibm.ws.concurrent.mp.ContextualConsumer;
-import com.ibm.ws.concurrent.mp.ContextualExecutor;
-import com.ibm.ws.concurrent.mp.ContextualFunction;
-import com.ibm.ws.concurrent.mp.ContextualRunnable;
-import com.ibm.ws.concurrent.mp.ContextualSupplier;
-import com.ibm.ws.concurrent.mp.ManagedCompletableFuture;
-import com.ibm.ws.concurrent.mp.ManagedCompletionStage;
-import com.ibm.ws.concurrent.mp.UnusableExecutor;
+import com.ibm.ws.concurrent.mp.spi.ContextFactory;
```

Comment:
Copyright date should probably be updated?

Reply:
That is the plan Tom.  I am just trying to get things all working first.

Reply:
Updated now that I have things working the way I want it to work.